cypress/e2e/helpers/surveyHelpers.ts

```diff
@@ -0,0 +1,93 @@
+import "moment/min/locales";
+import { format } from "date-fns";
+
+import * as surveyStrings from "../strings/surveyStrings";
+
+export const SERISD001_BODY_WEIGHT_ID: string =
+  '[data-testid="SURVEY_CARD_SER-ISD-001 - Body Weight"]';
+export const DATE_TEXT: string = "p.MuiTypography-body1.css-xhbppr";
+export const BUTTON: string = "button";
+export const VALUE_LBS_INPUT: string = "input.MuiInputBase-input";
+export const LBS_TEXT: string = "p.MuiTypography-root.css-10o8u6h";
+export const PROGRESS_BAR_50_IMG: string =
+  'span[role="progressbar"][aria-valuenow="50"]';
+export const PROGRESS_BAR_100_IMG: string =
+  'span[role="progressbar"][aria-valuenow="100"]';
+export const VALUE_PROGRESS_BAR_TEXT: string = "p.MuiTypography-root";
+export const PRIMARY_BUTTON: string = "button.MuiButton-containedPrimary";
+
+export const SNACK_BAR_ALERT_ID: string = '[data-testid="SNACK_BAR_ALERT"]';
+
+export const SCHEDULE_ONCE = (date: string, time: string) =>
+  `Once on ${date} at ${time}`;
+export const SCHEDULE_MONTHLY = (date: string, time: string) =>
+  `Once on ${date} at ${time}`;
+export const SCHEDULE_DAILY = (time: string) => `Daily at ${time}`;
+export const SCHEDULE_BIWEEKLY = (day: string, time: string) =>
+  `Biweekly on ${day} at ${time}`;
+export const SCHEDULE_TRIWEEKLY = (day: string, time: string) =>
+  `Biweekly on ${day} at ${time}`;
+export const SCHEDULE_WEEKLY = (day: string, time: string) =>
+  `Biweekly on ${day} at ${time}`;
+
+export const verifySurveyCard = (
+  surveyName: string,
+  frequency: string,
+  schedule: string,
+) => {
+  cy.contains("h6", surveyName);
+  cy.contains("h6", frequency);
+  cy.contains("h6", schedule);
+};
+
+export const verifyWeightScreenContent = () => {
+  cy.contains(BUTTON, surveyStrings.CLOSE);
+  cy.contains("h4", surveyStrings.SERISD001_BODY_WEIGHT);
+
+  const currentDate = new Date();
+  const formattedDateText = format(currentDate, "EEEE, MMMM dd");
```

Comment:
Just a suggestion as you won't need to access currentDate and it's short enough to put directly inside the funtion
```suggestion
  const formattedDateText = format(new Date(), "EEEE, MMMM dd");
```

Reply:
I had that to create this variable and add in the function because the vscode it was showed error.